### PR TITLE
React capstone project - Metrics webapp

### PR DESCRIPTION
**Pull request by Yothu**
# Description
Created a web application that gets numeric data from an API that can be filtered, the selected API is the [Narrativa API](https://covid19tracking.narrativa.com/index_en.html). It consists of a _home page_ and several _details pages_.
## Changes
- [x] Use **ES6 syntax**.
- [x] Use **ES6 modules**.
- [x] Use **React** and **Redux**.
- [x] Follow [GitFlow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md).
- [x] The data retrieved from the API should be stored in the Redux store.
- [x] Every page should have a unique route within the SPA.
- [x] The project should be deployed and accessible online.
- [x] Along with the category name, you displays some numeric values per category that come from the API.
- [x] When the user clicks (or taps) on a category item, the application navigates to the details page.
- [x] When the user clicks on the "Back" button (<), the user navigates to the home page.
- [x] Created unit tests for pure functions in the Redux code.
- [x] Created integration tests for the application using the React Testing Library. 
## Usage
+ Check the [README.md](https://github.com/Yothu/covid-19-monitor#readme) to know how to install and use this project.
+ Check the [video](https://www.loom.com/share/a7642682393e48adb885394b1986e05b) of me explaining the project functionalities.
+ Check the [live demo](https://covid-19-monitor2.netlify.app/) of this project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,73 +1,78 @@
...
+👤 David Vergaray
 
-## Acknowledgments
+- GitHub:   [@Yothu](https://github.com/Yothu)
+- Twiter:   [@Daivhy](https://twitter.com/Daivhy)
+- LinkedIn: [David Vergaray](https://www.linkedin.com/in/david-vergaray-almontes-051a11127/)
 
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
```
Comment (bimbolabuari) [inline README.md]: As required, Kindly add an "Acknowledgements" section that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)). 👍

File: src/redux/configureStore.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import { createStore, combineReducers, applyMiddleware } from 'redux';
+import logger from 'redux-logger';
+import thunk from 'redux-thunk';
+import regions from './regions/regions';
+// import missionsReducer from './missions/missions';
+
+const reducer = combineReducers({
+  regions,
+  // missionsReducer,
```
Comment (bimbolabuari) [inline src/redux/configureStore.js]: Let's remove commented codes, Kindly check for other occurrences 🙂

**Review by bimbolabuari — CHANGES_REQUESTED** [review]
Hi @Yothu, 

You have done a good job so far 🚀  

## Highlights 🎉 

- Correct Gitflow
- App has integration tests using React Testing Library
- The project contains a `Home` page and a `Detail` page.
- Deploys app with Netlify

There are some issues that you still need to work on to go to the next project but you are almost there! 🎉 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

--------


_Reviewed by  @bimbolabuari_

File: src/redux/configureStore.js (hunk 2 above)
Comment (Yothu) [reply src/redux/configureStore.js]: Just deleted all the comments, thanks for the review!

File: README.md (hunk 1 above)
Comment (Yothu) [reply README.md]: I forgot about it! Thanks for the reminder!

**Comment by omar-labana** [comment]
# **Approved ✔️**

Hi @Yothu,

Congratulations on completing this project! 👏🏼 🎉

You have correctly implemented all of the requirements📃 correctly and without any errors⚠️ or bugs🪲.

Cheers and good luck in your next milestones! 🍀

Feel free to leave any questions or comments in the PR thread(don't forget to tag me 😄) if something is not 100% clear.

---
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**